includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -127,6 +123,13 @@ public function __construct(
 				'description' => '',
 				'default'     => 'no',
 			],
+			'cc_enabled'                          => [
+				'title'       => __( 'Enable/disable Credit Card', 'woocommerce-payments' ),
+				'label'       => __( 'Enable WooCommerce Payments Credit Card', 'woocommerce-payments' ),
+				'type'        => 'checkbox',
+				'description' => '',
+				'default'     => 'no',
```

Comment:
Probably this should default to 'yes'. We definitely want it enabled for stores which already have been using CC when it was the only payment method. Probably we want to automatically enable CC for any stores which have just installed WooCommerce Payments.

Reply:
Good catch! The existing `woocommerce_payments` settings inside `wp_options` would not have this enabled, thus it would disabling credit card as a payment method. 

Changing this to `yes`.